checkdigit/crc.py

```diff
@@ -0,0 +1,74 @@
+# /usr/bin/env python
+# This file is part of checkdigit.
+
+# checkdigit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# checkdigit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with checkdigit.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Cyclic Redundancy Check.
+
+A block of binary as a form of data validation is appended to a bitstring.
+This can be easily checked even by very simple circuitry,
+and can also be used to correct some errors.
+
+WARNING: THIS IS NOT A FAST IMPLEMENTATION OF CRC.
+
+If you want a fast implementation look elsewhere.
+
+For more information, please look at the wiki page for this module:
+https://add.wiki.page.here
```

Comment:
```suggestion
```

Since we've moved to [RTD](https://checkdigit.readthedocs.io/en/latest/), I've removed the wiki links from the other files https://github.com/harens/checkdigit/commit/591cc3853cec2a4a5713c43fcffe375ca1ddb0cb. In line with this, would it be possible to remove these lines? Thanks.